libbeat/otelbeat/converters/testdata/supported.yml

```diff
@@ -0,0 +1,35 @@
+receivers:
+  filebeatreceiver:
+    filebeat:
+      inputs:
+        - enabled: true
```

Comment:
You need a test with more than one input.

On that note, it looks to me like we would create a single Beat receiver with multiple inputs in this case. I am curious if this will turn out to have performance limitations later on, but this is something we'd need to specifically benchmark to see if there is a difference.

Reply:
@khushijain21 I created the task here and added to the sprint https://github.com/elastic/ingest-dev/issues/4754